src/main/java/apim/restful/importexport/utils/APIImportUtil.java

```diff
@@ -505,23 +503,22 @@ private static void addAPIWsdl(String pathToArchive, API importedApi, String cur
      */
     private static void addSwaggerDefinition(APIIdentifier apiId, String archivePath)
             throws APIImportException {
+
         try {
             String swaggerContent = FileUtils.readFileToString(
                     new File(archivePath + APIImportExportConstants.SWAGGER_DEFINITION_LOCATION));
             provider.saveSwagger20Definition(apiId, swaggerContent);
         } catch (APIManagementException e) {
             log.error("Error in adding Swagger definition for the API. ", e);
-            throw new APIImportException("Error in adding Swagger definition for the API. " +
-                    e.getMessage());
+            throw new APIImportException("Error in adding Swagger definition for the API. " + e.getMessage());
```

Comment:
throwiing the exception only with the exception message. Any reason whole exception not throwing